translations/fi/LC_MESSAGES/circulation-admin.po

```diff
@@ -2,669 +2,705 @@
+"Last-Translator: Marika Kaskinen, 2024\n"
+"Language-Team: Finnish (https://app.transifex.com/national-library-of-finland/teams/183401/fi/)\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
+"Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.13.0\n"
+"Language: fi\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
 #: api/admin/announcement_list_validator.py:35
 #, python-format
 msgid "Invalid announcement list format: %(announcements)r"
-msgstr ""
+msgstr "Virheellinen ilmoitusluettelon muoto: %(announcements)r"
 
 #: api/admin/announcement_list_validator.py:50
 #, python-format
 msgid "Too many announcements: maximum is %(maximum)d"
-msgstr ""
+msgstr "Liian monta ilmoitusta: enimmäismäärä on %(maximum)d"
 
 #: api/admin/announcement_list_validator.py:62
 #, python-format
 msgid "Duplicate announcement ID: %s"
-msgstr ""
+msgstr "Kaksi ilmoitustunnusta: %s"
 
 #: api/admin/announcement_list_validator.py:71
 #, python-format
 msgid "Invalid announcement format: %(announcement)r"
-msgstr ""
+msgstr "Virheellinen ilmoitusmuoto: %(announcement)r"
 
 #: api/admin/announcement_list_validator.py:85
 #, python-format
 msgid "Invalid announcement ID: %(id)s"
-msgstr ""
+msgstr "Virheellinen ilmoitustunnus: %(id)s"
 
 #: api/admin/announcement_list_validator.py:95
 #, python-format
 msgid "Missing required field: %(field)s"
-msgstr ""
+msgstr "Pakollinen kenttä puuttuu: %(field)s"
 
 #: api/admin/announcement_list_validator.py:141
 #, python-format
 msgid "Value too short (%(length)d versus %(limit)d characters): %(value)s"
-msgstr ""
+msgstr "Liian lyhyt arvo (%(length)d vs. %(limit)d merkkiä): %(value)s"
 
 #: api/admin/announcement_list_validator.py:153
 #, python-format
 msgid "Value too long (%(length)d versus %(limit)d characters): %(value)s"
-msgstr ""
+msgstr "Liian pitkä arvo (%(length)d vs. %(limit)d merkkiä): %(value)s"
 
 #: api/admin/announcement_list_validator.py:188
 #, python-format
 msgid "Value for %(field)s is not a date: %(date)s"
-msgstr ""
+msgstr "Kentän %(field)s arvo ei ole päivämäärä: %(date)s"
 
 #: api/admin/announcement_list_validator.py:201
 #, python-format
 msgid "Value for %(field)s must be no earlier than %(minimum)s"
-msgstr ""
+msgstr "Kentän %(field)s arvo saa olla aikaisintaan %(minimum)s"
 
 #: api/admin/problem_details.py:9
 msgid "Admin auth not configured"
-msgstr ""
+msgstr "Ylläpitäjän todennusta ei ole määritetty"
 
 #: api/admin/problem_details.py:10
-msgid "This circulation manager has not been configured to authenticate admins."
-msgstr ""
+msgid ""
+"This circulation manager has not been configured to authenticate admins."
+msgstr "Tätä lainauksen hallintaa ei ole määritetty todentamaan ylläpitäjiä."
 
 #: api/admin/problem_details.py:16
 msgid "Admin auth mechanism not configured"
-msgstr ""
+msgstr "Ylläpitäjän todennusmekanismia ei ole määritetty"
 
 #: api/admin/problem_details.py:17
 msgid ""
```

Comment:
Some of the translations are missing `msgid`. Looks like the translation scripts are not parsing these correctly when the `_()` method is formatted to multiple lines:
```
_(
    "Value to be translated"
),
```

Reply:
Nevermind, the id is just on multiple lines 🤦